CHANGELOG.md

```diff
@@ -21,7 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fixed missing name for CVE results on result detailspage [#2892](https://github.com/greenbone/gsa/pull/2892)
 - Fixed setting secret key in RADIUS dialog [#2891](https://github.com/greenbone/gsa/pull/2891)
 - Fixed setting result UUID in notes dialog [#2889](https://github.com/greenbone/gsa/pull/2889)
-
+- Fixed setting whether to include related resources for new permissions [#2931](https://github.com/greenbone/gsa/pull/2891)
```

Comment:
@bjoernricks @greenbone/gsa-maintainers Seems those entries where unrelated to this MR and at least the RADIUS entry is now duplicated in the changelog (similar happened in #2957 as well).

Reply:
Fixes in #2991 and #2992